data/importers/page.md

```diff
@@ -200,24 +200,175 @@ For each importer, we will cover only columns of interest.
 * The ICAO or IATA code for the arrival airport.
   *Note: you should only use one of ID, ICAO or IATA when adding routes and the Import modal will ask which you wish to use. If you're using the ID, this field should be blank*
 ---
-* Start Date / End Date
-* Date and Time when the route starts/ends. End Date can be blank; Start Date can be any day in the past or the future. Format for both fields is yyyy-mm-dd hh:mm - please ensure you set correct formatting in your spreadsheet editor.
+* Callsign
+* Callsign for the route in the ICAO format. 
+  *Note: Must be a valid combination of ICAO and IATA code from the Airline Callsign Parameters section in the Orwell Settings -> VDS*
+---
+* Flight Number
+* Flight Number for the route in the IATA format. 
+  *Note: Must be a valid combination of ICAO and IATA code from the Airline Callsign Parameters section in the Orwell Settings -> VDS*
 ---
 * Fleets
 * Comma separated list of Fleet IDs - 5, 15, 25, 30. IDs can be retrieved from Export or VDS.
 ---
+* Altitude
+* Cruise altitude in the feet format (e.g. 35000).
+*Note: Can be Zero, in case you want the Simbrief to decide the cruise altitude*
+---
+* Start Date / End Date
+* Date and Time when the route starts/ends. End Date can be blank; Start Date can be any day in the past or the future. Format for both fields is yyyy-mm-dd hh:mm - please ensure you set correct formatting in your spreadsheet editor.
+---
+* CI
+* Cost Index for the flight. Can be any value from 0 to 999.
+*Note: Can be blank. In that case, if using the Dispatch via Simbrief, an AUTO value is assigned to determine the Cost Index value for the flight, which is handled by Simbrief*
+---
+* Route
+* Route for the flight. vAMSYS will get rid of any SIDs, STARs, and Step Climbs within the route
+*Note: Can be blank, but the column is required. In that case, if using the Dispatch via Simbrief, the Simbrief will assign the route.*
+---
+* Departure Time
+* Standard departure time of the flight, in hh:mm format.
+*Note: Can be blank, but the column is required*
+---
+* Arrival Time
+* Standard arrival time of the flight, in hh:mm format.
+*Note: Can be blank, but the column is required*
+---
+* Flight Length
+* Flight length in hh:mm format.
+*Note: Can be blank, but the column is required. If blank, the value is calculated by vAMSYS by a general equation for an airliner*
+---
+* Flight Distance
+* Flight distance in nautical miles.
+*Note: Can be blank, but the column is required. If blank, the value is calculated by vAMSYS based on the route*
```

Comment:
Would change it to say that if empty, the distance is calculated by vAMSYS as a great circle distance